v23.1/connect-to-a-changefeed-kafka-sink-with-oauth-using-okta.md

```diff
@@ -0,0 +1,291 @@
+
+An overview of the workflow involves:
+
+1. Creating an Okta application integration and authorization server scope.
+1. Configuring the Kafka cluster.
+1. Streaming the changefeed to the Kafka cluster.
+
+## Before you begin 
+
+You will need the following set up before starting this tutorial:
+
+- A CockroachDB cluster. You can use a {{ site.data.products.db }} or {{ site.data.products.core }} cluster. If you are using {{ site.data.products.serverless }} or {{ site.data.products.dedicated }}, see the [Quickstart with CockroachDB](../cockroachcloud/quickstart.html) guide. For {{ site.data.products.core }} clusters, see the [install](install-cockroachdb-mac.html) page. 
+- An [Okta Developer account](https://developer.okta.com/signup/).
+- (Optional) A Kafka cluster. This tutorial includes [Kafka cluster setup](#step-2-create-a-kafka-sink). Note that this tutorial was tested with Kafka [version 2.8.2](https://kafka.apache.org/downloads). The Kafka cluster configuration may vary with different versions, but the Okta setup and changefeed creation will be the same. 
+
+This tutorial uses the Cockroach Labs [`movr`](movr.html) workload as an example database.
+
+## Step 1. Create an Okta application and scope
+
+In this step, you will create an application integration in your Okta developer account and set up your [authorization server](https://help.okta.com/oie/en-us/Content/Topics/Security/api-build-oauth-servers.htm) with a [scope](https://help.okta.com/oie/en-us/Content/Topics/Security/api-config-scopes.htm?cshid=create-scopes). The authorization server produces the OAuth 2.0 tokens the changefeed will use to connect to the Kafka sink. Scopes contained within an authorization server provide a way to limit access that operations have to the OAuth 2.0 access tokens.
+
+1. Log in to your Okta Developer account and navigate to **Applications** in the sidebar.
+
+    In the dropdown, click **Applications** and then **Create App Integration**.
+
+    On the **Create a new app integration** menu, select **API services** for OAuth 2.0 access tokens. Click **Next**.
+
+    Add an **App integration name** and then **Save**.
+
+    Once you have created the application integration, you'll create a scope for your authorization server.
+
+1. Navigate to **Security** in the sidebar and then **API**. From here, you can add a new authorization server, or use the default authorization server. This tutorial will use the default server.
+
+    Click on the name of your authorization server and then select the **Scopes** tab. 
+
+    Select **Add Scope**:
+    - Add `kafka` as the name for your scope and include an optional description. 
+    - For **User Consent**, leave the default **Implicit** selected. 
+    - Check the box to set this as the **Default Scope**. 
+    - Optionally check **Include in public metadata** if you wish to include this scope in public metadata.
+
+    Click **Create**.
+
```

Comment:
This might be easier to follow if you split it into two headings, enabling more numbered steps within each, e.g.

```
## Step 1. Create an Okta application
[...]
## Step 2. Configure a scope for your Okta application
```

Reply:
Adapted per this comment

Reply:
Looks great